<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editCCarDialog</class>
 <widget class="QDialog" name="editCCarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit company car</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_22">
    <item>
     <widget class="QLabel" name="label_47">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="model_CC_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_61">
      <property name="text">
       <string>Company Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="comname_CC_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_63">
      <property name="text">
       <string>Organization Number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="orgnr_CC_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_64">
      <property name="text">
       <string>Ensured by:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ensur_CC_2"/>
    </item>
   </layout>
  </widget>
  <zorder>buttonBox</zorder>
  <zorder>layoutWidget_14</zorder>
  <zorder>label_64</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editCCarDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editCCarDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
